### DISCUSSION TITLE
How to use "Import Blogs List" button?

### DISCUSSION THREAD

**Discussion by thomas694** [TumblThreeApp/TumblThree#379] [Q&A (Question & Answer)]
The "Import Blogs List" button - What is the name of the file that contains this data, and where is it? Been searching for this for days. Would appreciate your assistance.

Thank you :)

_Originally posted by @1cabron in https://github.com/TumblThreeApp/TumblThree/issues/323#issuecomment-1301473080_

**Comment by thomas694** [comment]
@1cabron
TumblThree doesn't provide a list of blogs. You can bulk import blogs by creating a text file with one blog url per line and loading it with the "Import Blog List" button.

**Reply by 1cabron** [reply]
Q.E.D - quite easily done 

Thank you for your quick response. :)

**Comment by 1cabron** [comment]
Thank you very much. I have been using tumblThree for many years. Great tool, and I truly appreciate your response.

 

Take care, and keep up the good work.

😊
